src/aioswitcher/api/messages.py

```diff
@@ -130,16 +131,22 @@ def get_thermostat_remote_id(self) -> str:
         remote_hex = unhexlify(self._hex_response)
         return remote_hex[84:92].decode().rstrip("\x00")
 
-    def get_shutter_position(self) -> int:
-        """Return the current shutter position."""
-        hex_pos = self._hex_response[152:154].decode()
+    def get_shutter_position(self, index: int) -> int:
+        """Return the current position of the shutter 0 <= pos <= 100."""
+        index -= 1
```

Comment:
```suggestion
```

Reply:
Honestly I don't agree with this change.
You have in your house zero and one shutters?
No you have one and two.
I want go and get shutter number one and shutter number 2.
Using 0 and 1 make no sense here..

Using 0 as a starting point makes sense for indexing that outside of human logic mainly for array, but here you are talking about the shutter number you want to control.
If you buy one Switcher Runner you can control shutter with id 1 which is one shutter you have.

Reply:
Why does it have to be related to what the user see? User can still see the first shutter as 1, it would probably mean that we will have one or two places that we have to add one when we display it to the user, but it doesn't make sense that all low level APIs will just reduce 1 from the input, they are not exposed to the user

Reply:
I agree with @thecode on this. Starting an indexed list from 1 instead of 0 might be in line with the user's perspective, but this is error-prone. A contributor might add some logic and look for the 0 index member. 

Also, changing the value of an argument from the function's signature is a bad practice unless the function is meant to be used recursively. But this is irrelevant if you switch to zero-based indexing and dump the `index -= 1` statement.

Reply:
Change has been done + added testes